homeassistant/components/zwave_me/__init__.py

```diff
@@ -0,0 +1,126 @@
+"""The Z-Wave-Me WS integration."""
+import asyncio
+import logging
+
+from zwave_me_ws import ZWaveMe, ZWaveMeData
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.dispatcher import async_dispatcher_connect, dispatcher_send
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, PLATFORMS, ZWAVE_PLATFORMS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def get_uuid(url, token):
+    """Get an uuid from Z-Wave-Me."""
+    conn = ZWaveMe(url=url, token=token)
+    uuid = None
+    if await conn.get_connection():
+        uuid = await conn.get_uuid()
+    await conn.close_ws()
+    return uuid
+
+
+async def async_setup_entry(hass, entry):
+    """Set up Z-Wave-Me from a config entry."""
+    hass.data[DOMAIN] = {}
```

Comment:
Since we allow more than one config entry, we need to use dict.setdefault or similar to avoid overwriting the key in hass.data.